server/raft.go

```diff
@@ -1269,8 +1270,18 @@ func (n *raft) isCurrent(includeForwardProgress bool) bool {
 		return false
 	}
 
+	// If we were previously logging about falling behind, also log when the problem
+	// was cleared.
+	clearBehindState := func() {
+		if n.hcbehind {
+			n.warn("Health check OK, no longer falling behind")
+			n.hcbehind = false
+		}
+	}
+
 	// Make sure we are the leader or we know we have heard from the leader recently.
 	if n.State() == Leader {
+		clearBehindState()
```

Comment:
This not only applicable to leaders no?

Reply:
I put this here just to cover the case of if you became leader in that time, to ensure the behind state gets cleared.

Reply:
Gotcha..